### PR TITLE
Structured Logging migration: for package pkg/scheduler/internal/cache

### PR DESCRIPTION
**Pull request by sanwishe**
Signed-off-by: sanwishe <jiang.mingzhi35@zte.com.cn>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Structured Logging migration: migrate package pkg/scheduler/internal/cache/
xref:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
"NONE"
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
"NONE"
```

### CODE REVIEW

File: pkg/scheduler/internal/cache/node_tree.go [hunk 1]
```diff
@@ -76,13 +76,13 @@ func (nt *nodeTree) removeNode(n *v1.Node) error {
 				if len(nt.tree[zone]) == 0 {
 					nt.removeZone(zone)
 				}
-				klog.V(2).Infof("Removed node %q in group %q from NodeTree", n.Name, zone)
+				klog.V(2).Infof("Removed node from NodeTree", "node", n.Name, "group", zone)
```
Comment (yangjunmyfm192085) [inline pkg/scheduler/internal/cache/node_tree.go:79]:
```suggestion
				klog.V(2).InfoS("Removed node from NodeTree", "node", n.Name, "group", zone)
```

**Comment by pacoxu** [comment]
dup with #98135?

**Comment by Huang-Wei** [comment]
> dup with #98135?

#98135 was raised earlier than this one, so I left https://github.com/kubernetes/kubernetes/pull/98135#issuecomment-909446209 to see if the author is going to finish that one.

**Comment by dims** [comment]
Is this PR still needed, please rebase if so (or we can close it?)

**Comment by Huang-Wei** [comment]
The proposed changes have been covered in another PR.

/close

**Pull request closed without merging**